audio/speech/getting-started/get_started_with_chirp_2_sdk.ipynb

```diff
@@ -0,0 +1,1331 @@
+        "def evaluate_stt(\n",
+        "    actual_transcriptions: list[str],\n",
+        "    reference_transcriptions: list[str],\n",
+        "    audio_sample_file_uri: str = INPUT_LONG_AUDIO_SAMPLE_FILE_URI,\n",
+        ") -> pd.DataFrame:\n",
+        "    \"\"\"\n",
+        "    Evaluate speech-to-text (STT) transcriptions against reference transcriptions.\n",
+        "    \"\"\"\n",
+        "    audio_uris = [audio_sample_file_uri] * len(actual_transcriptions)\n",
+        "    evaluations = []\n",
+        "    for audio_uri, actual_transcription, reference_transcription in zip(\n",
+        "        audio_uris, actual_transcriptions, reference_transcriptions\n",
+        "    ):\n",
+        "        evaluation = {\n",
+        "            \"audio_uri\": audio_uri,\n",
+        "            \"actual_transcription\": actual_transcription,\n",
+        "            \"reference_transcription\": reference_transcription,\n",
+        "            \"wer\": jiwer.wer(reference_transcription, actual_transcription),\n",
+        "            \"cer\": jiwer.cer(reference_transcription, actual_transcription),\n",
+        "        }\n",
+        "        evaluations.append(evaluation)\n",
+        "\n",
+        "    evaluations_df = pd.DataFrame(evaluations)\n",
+        "    evaluations_df.reset_index(inplace=True, drop=True)\n",
+        "    return evaluations_df\n",
+        "\n",
+        "\n",
+        "def plot_evaluation_results(\n",
+        "    evaluations_df: pd.DataFrame,\n",
+        ") -> go.Figure:  # Return type is go.Figure\n",
+        "    \"\"\"\n",
+        "    Plot the mean Word Error Rate (WER) and Character Error Rate (CER) from the evaluation results.\n",
+        "    \"\"\"\n",
+        "    mean_wer = evaluations_df[\"wer\"].mean()\n",
+        "    mean_cer = evaluations_df[\"cer\"].mean()\n",
+        "\n",
+        "    trace1 = go.Bar(x=[\"WER\", \"CER\"], y=[mean_wer, mean_cer], name=\"Mean Error Rate\")\n",
+        "\n",
+        "    trace2 = go.Scatter(\n",
+        "        x=[\"WER\", \"CER\"], y=[0.5, 0.5], mode=\"lines\", name=\"Baseline (0.5)\"\n",
+        "    )\n",
+        "\n",
+        "    layout = go.Layout(\n",
+        "        title=\"Speech-to-Text Evaluation Results\",\n",
+        "        xaxis=dict(title=\"Metric\"),\n",
+        "        yaxis=dict(title=\"Error Rate\", range=[0, 1]),\n",
+        "        barmode=\"group\",\n",
+        "    )\n",
+        "\n",
+        "    fig = go.Figure(data=[trace1, trace2], layout=layout)\n",
+        "    return fig"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "VPVDNRyVxquo"
+      },
+      "source": [
+        "## Transcribe using Chirp 2\n",
+        "\n",
+        "You can use Chirp 2 to transcribe audio in streaming, real-time and batch modes. In the following sections, you explore how to use the Speech-to-Text v2 API to transcribe audio in these three different scenarious."
```

Comment:
Could you add a brief description for each of these modes and their use cases? (Bullet points could be easier to read)

Reply:
done

Reply:
done